pkg/operators/wasm/testdata/map/program.go

```diff
@@ -128,6 +129,48 @@ func gadgetStart() int {
 		return 1
 	}
 
+	mapSpec := api.MapSpec{
+		Name:       "map_test",
+		Type:       api.Hash,
+		KeySize:    uint32(4),
+		ValueSize:  uint32(4),
+		MaxEntries: 1,
+	}
+
+	newMap, err := api.NewMap(mapSpec)
+	if err != nil {
+		api.Errorf("creating map %s", mapSpec.Name)
+	}
+	defer newMap.Close()
```

Comment:
Could you add tests:
- Running Close several times
- Trying Lookup/Put after Close and checking the error code